packages/core/src/util/renderer.ts

```diff
@@ -375,7 +375,7 @@ export const mapStateToControlProps = (
   const visible = has(ownProps, 'visible')
     ? ownProps.visible
     : isVisible(uischema, rootData, ownProps.path);
-  const enabled = has(ownProps, 'enabled')
+  const enabled = (has(ownProps, 'enabled') && ownProps.enabled !== undefined)
```

Comment:
It seems to me that the code was written with the expectation that `has` also checks for `undefined`. As this is not the case we should use the `foo !== undefined` check everywhere instead of `has`. Please replace all usages of `has` with an undefined check in all files in a separate commit.